### PR TITLE
Project documentation - Ana Maria Comeaga

### PR DESCRIPTION
**Pull request by anamaria2004**
### Pull Request Overview

This pull request adds the project documentation for FloraMood Smart Pot.

### Testing Strategy

This pull request was tested by Ana-Maria Comeaga.

### TODO or Help Wanted

I managed to review the asked changes.

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Ana-Maria Comeaga <anamariacomeaga@yahoo.com>

### CODE REVIEW

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/Smart Pot.zip
Comment (alexandruradovici) [inline docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/Smart Pot.zip:1]: Place these in your project repository, each file separate, not zipped.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Please fill the pull request template.

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/Smart Pot.zip
Comment (anamaria2004) [reply docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/Smart Pot.zip:1]: I solved the issue. Thank you!

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/your_curs_upb_ro_id.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+| [Watering Pump] | Watering the system | [26.13 RON](https://www.emag.ro/pompa-de-apa-kruger-meier-sturmkraft-220jk-x1-220l-h-3-8-x-3-7-x-2-8-cm-negru-5903900042987/pd/DTFJ05MBM/?X-Search-Id=7b217b870d8e7e7ff1d5&X-Product-Id=113031364&X-Search-Page=1&X-Search-Position=32&X-Section=search&X-MB=0&X-Search-Action=view) |
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/your_curs_upb_ro_id.md:93]: Do you need these two libraries?

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/your_curs_upb_ro_id.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+On the other hand, taking into consideration the growth of concerns about the environmental sustainability and conservation, a smart pot is a good way of promoting responsible plan care practices. Through the optimization of resource usage and minimization of waste, this project aligns with the goal of reducing the environmental impact and helping the nature.
+
+## Architecture 
+
+The FloraMood Smart Pot is powered up by a battery. Then, it collects the parameters from the sensors, like humidity, temperature, and light. Based on them and on their thresholds, a specific emoji is displyed on the LCD Module, in order to transmit the state in which the plant is on that moment. The watering system is using a simple water pump with a gear, that automatically powers up when the humidity level is under the set threshold and stops after giving the plant the necessary ammount of water, and the humidity level comes back in its normal range.
+
+The diagram depicts the functioning of FloraMood Smart Pot. Here is a breakdown of its components:
```
Comment (alexandruradovici) [inline docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/your_curs_upb_ro_id.md:31]: The diagram should be displayed here.

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/Architecture.png
Comment (alexandruradovici) [inline docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/Architecture.png:1]: Add this file to your project repository.

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/obj.mtl
Comment (alexandruradovici) [inline docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/obj.mtl:1]: Add this file to your project repository.

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/your_curs_upb_ro_id.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
+| [ads1x1x](https://github.com/vgasparyan/ads1256-rs) | Provides drivers for ADS1x1x ADCs | Used to interact with the ADS1115 ADC |
+| [std](https://github.com/rust-lang/rust/blob/master/library/std/src/lib.rs) | Standard library for Rust | Provides core functionality and data structures |
+| [embedded-hal](https://github.com/embassy-rs/embassy) | Hardware abstraction layer for embedded systems | Used for abstracting hardware interactions |
+| [rppal](https://github.com/golemparts/rppal) | Raspberry Pi Pico hardware interface library | Interface with Raspberry Pi Pico hardware components |
+| [lcd_2inch](https://github.com/zadi15/picoLCD) | Library for interfacing with 2-inch LCD displays | Initializing and controlling the LCD display |
+| [thread](https://github.com/rust-lang/rust/blob/master/library/std/src/thread/mod.rs) | Thread management library | Creating and managing threads |
```
Comment (alexandruradovici) [inline docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/your_curs_upb_ro_id.md:99]: I think this is from the standar library and is not available for embedded systems.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/your_curs_upb_ro_id.md (hunk 1 above)
Comment (anamaria2004) [reply docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/your_curs_upb_ro_id.md:93]: I need them for displaying the specific emoji on the LCD screen.

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/obj.mtl
Comment (alexandruradovici) [inline docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/obj.mtl:1]: Please move this file to your project repository.

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/tinker.obj
Comment (alexandruradovici) [inline docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/tinker.obj:1]: Please move this file to your project repository.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please rename the folder to your username.

File: docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/obj.mtl
Comment (anamaria2004) [reply docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/obj.mtl:1]: I think that I solved it, and now it should be okay.

Comment (alexandruradovici) [reply docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/obj.mtl:1]: The file is still there. If you used the commit changes button, make sure you use `git pull` to update your local branch.

Comment (anamaria2004) [reply docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/obj.mtl:1]: I also tried the git pull command, but the files still seem to be there. I also put this into the file: Architecture photo

Comment (alexandruradovici) [reply docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/obj.mtl:1]: Just delete the files form you branch and make sure you commit the deletions. To make sure, run in the command line:

```
git add .
git commit -am "deleted files"
git push
```

Comment (anamaria2004) [reply docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/obj.mtl:1]: I modified the files. Now, it is okay?

**Review by alexandruradovici — APPROVED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**